docs/api.md

```diff
@@ -8,16 +8,27 @@ Use auro-accordion-group if you want to have auto closing accordion components w
 | Property     | Attribute    | Type      | Default | Description                                      |
 |--------------|--------------|-----------|---------|--------------------------------------------------|
 | `alignRight` | `alignRight` | `Boolean` |         | If set, the trigger content will align right.    |
-| `chevron`    | `chevron`    | `Boolean` |         | If set, the accordion trigger will have a chevron showing expanded/collapsed state. |
+| `emphasis`   | `emphasis`   | `Boolean` |         | If set, emphasis styles will be applied to the auro-accordions. |
 | `expanded`   | `expanded`   | `Boolean` | false   | If set, the accordion is expanded.               |
 | `fluid`      | `fluid`      | `Boolean` |         | If set, the trigger and content will be 100% width. |
+| `group`      | `group`      | `Boolean` |         | Attribute will be set on accordion when it appears in an accordion group. |
```

Comment:
This attribute allows us to style accordions and the components in their shadowDOM when the accordions are within a group. I was not able to apply the styles necessary to the children of the accordion because you cannot use the `::part` selector in conjuction with the `::slotted` selector.

That being said, I have updated the name of this attribute from `group` to `grouped`.